addons/website_event_meet/controllers/community.py

```diff
@@ -138,7 +138,7 @@ def _event_meeting_room_page_get_values(self, event, meeting_room):
         # search for meeting room list
         meeting_rooms_other = request.env['event.meeting.room'].sudo().search([
             ('event_id', '=', event.id), ('id', '!=', meeting_room.id), ('is_published', '=', True),
-        ])
+        ], limit=6)
```

Comment:
Could be nice to add a small docstring explaining this limit (I suppose its to limit the number of results displayed in the template)